docs/guides/platform/accounts/2fa/index.md

```diff
@@ -0,0 +1,146 @@
+
+- [1Password](https://1password.com/)
+- [Bitwarden](https://bitwarden.com/)
+- [Keeper](https://www.keepersecurity.com/)
+
+The primary downsides of using your password manager as your OTP provider are security and cost. If a malicious actor gains access to your password manager, they also now have access to your OTPs. To prevent this, consider using a dedicated application (see below).
+
+### Use a Dedicated Authenticator App
+
+There are quite a few free (and paid) third-party authenticator applications available. They are typically more secure than using your password manager's OTP functionality as a malicious actor cannot gain access to your Linode account (or any other 2FA protected account) unless they know your password and have access to the particular device on which the authenticator app is installed, typically your smartphone.
+
+- [Authy](https://authy.com/features/setup/)
+- [Duo Mobile](http://guide.duosecurity.com/third-party-accounts)
+- [Google Authenticator](http://support.google.com/accounts/bin/answer.py?hl=en&answer=1066447)
+- [Microsoft Authenticator](https://www.microsoft.com/en-us/security/mobile-authenticator-app)
+
+## Enabling 2FA
+
+Enable two-factor authentication to start using it with your Linode account.
+
+1.  Log in to the [Cloud Manager](https://cloud.linode.com).
+
+1.  Navigate to the Login & Authentication page of your profile by clicking on your **username** in the top right of the screen. Select **Login & Authentication** from the dropdown menu.
+
+    ![Click your username and select Login & Authentication](profile-link.png)
+
+1. Within the **Login Method** section, select **Linode** as the login provider. If you configure a third-party provider (such as Google or GitHub), you instead can manage 2FA directly through that provider and not through Linode.
+
+    ![Select the Login Method](tpa-options.png)
+
+1.  Under **Security Settings**, verify that you have configured all 3 security questions and then enable **Two-Factor Authentication (2FA)**.
+
+    ![Enable 2FA](tfa-enable.png)
+
+1.  A new form appears, displaying the QR code, secret key, and a section to enter a token.
+
+1.  Open your preferred 2FA provider on your smartphone or desktop. See [Choosing an Authenticator App](#choosing-an-authenticator-app).
+
+1.  Scan the QR code into the app or enter the secret key. This process will vary depending on your provider. Within Authy, tap **Add Account** and then tap **SCAN QR CODE**. Point your device's camera at the barcode on your computer screen. The app automatically creates a new token for your Cloud Manager login. It is labeled **LinodeManager:user**. Change the account name if necessary, and press **Done**.
```

Comment:
I think we need to set up somewhere that we are using Authy to demonstrate things, this mention seems out of the blue.

Reply:
Fair point. I removed the reference to Authy early on but can either add it back in or modify this workflow.